src/orders-update.js

```diff
@@ -64,38 +64,51 @@ export default class OrdersUpdate {
       })
   }
 
-  // Get the state ID from the API based on the key
-  // getStateReference :: String -> Promise -> String
-  getStateReference (key) {
+  // Get the ID from the API based on the key
+  // getStateReference :: (String, String, String) -> Promise -> String
+  getIdFromKey (key, typeId, endpoint) {
+    if (typeof key === 'object')
```

Comment:
Don't think this belongs inside this function, also I don't get why it's there. :/ Could you explain?